test/test_priority_stateful.py

```diff
@@ -0,0 +1,62 @@
+import random
+
+from hypothesis import note
+from hypothesis import strategies as st
+from hypothesis.stateful import RuleBasedStateMachine, rule
+
+import priority
+
+
+class PriorityQueue(RuleBasedStateMachine):
+
+    def __init__(self):
+        super(PriorityQueue, self).__init__()
+        self.tree = priority.PriorityTree()
+        self.stream_ids = [0]
```

Comment:
Heh, call me silly, but isn't this better used as a set? We don't care about ordering, so is there any advantage to the list?